redirects.yaml

```diff
@@ -328,6 +328,12 @@ wwwirishfreemaninfocom:
   sslname: 'irishfreemaninfo.com'
   ca: 'LetsEncrypt'
   disable_event: false
+wikirtapptk:
+  url: 'wiki.rtapp.tk'
+  additional_domain: 'wiki.recaptime.tk'
```

Comment:
What's this for? This is a very rare thing to need to use. It's currently only used for like 2 other domains which need wildcards I think. Also we need sslname set for this redirect I think.